code/modules/projectiles/guns/projectile/automatic/drozd.dm

```diff
@@ -1,22 +1,24 @@
 /obj/item/weapon/gun/projectile/automatic/drozd
 	name = "Excelsior .35 Auto \"Drozd\""
-	desc = "An excellent fully automatic submachinegun. Famous for it's perfomance in close quarters. Uses .35 Auto rounds."
+	desc = "An excellent fully automatic submachinegun. Made by \"Excelsior\" by reverse-engineering and improving upon Moebius \
+			reverse-engineered C20r robust design to sevelop even more deadly weapon. Famous for it's perfomance in close quarters. \
+			Has better than average fire rate. Uses .35 Auto rounds."
 	icon = 'icons/obj/guns/projectile/drozd.dmi'
 	icon_state = "drozd"
 	item_state = "drozd"
 	w_class = ITEM_SIZE_NORMAL
-	can_dual = 1
 	force = WEAPON_FORCE_PAINFUL
 	caliber = CAL_PISTOL
 	origin_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 2)
 	ammo_type = "/obj/item/ammo_casing/pistol"
 	load_method = MAGAZINE
 	mag_well = MAG_WELL_PISTOL|MAG_WELL_SMG
 	magazine_type = /obj/item/ammo_magazine/smg
-	matter = list(MATERIAL_PLASTEEL = 15, MATERIAL_PLASTIC = 3)
-	price_tag = 2000
-	damage_multiplier = 0.75
-	recoil_buildup = 1.5
+	matter = list(MATERIAL_PLASTEEL = 12, MATERIAL_STEEL = 4, MATERIAL_PLASTIC = 4)
```

Comment:
Individually those numbers make sense, together this gun is a meme mess.

Reply:
My following proposal is to make it more or less identical to C20r so it would be just excelsior skin of general purpose overall good SMG. If you want to nerf something in particular about it - point it out.

Reply:
Should be resolved now. Made it rare AP-focused SMG that you can only find on disk from advanced disk spawners for now. Later would add as very rare gun loot itself.